python/legion_linux/legion_linux/legion_gui.py

```diff
@@ -1717,7 +1712,7 @@ def main():
     # Start Windows
     if controller.model.app_model.automatic_close.get():
         main_window.close_after(3000)
-    main_window.show()
```

Comment:
Not make it Maximized it a bit annoying for people that open the app on login

But for everthing else nice job :)